openreview/conference/builder.py

```diff
@@ -723,7 +723,7 @@ def setup_post_submission_stage(self, force=False, hide_fields=[]):
             if self.submission_stage.second_due_date < now:
                 self.setup_final_deadline_stage(force, hide_fields)
         else:
-            if force or not self.submission_stage.due_date or self.submission_stage.due_date < datetime.datetime.now():
+            if force or self.submission_stage.due_date < datetime.datetime.now():
```

Comment:
What happens with due_date is None? 

I'm getting this error in a python console:

```

TypeError: '<' not supported between instances of 'NoneType' and 'datetime.datetime'

```

Reply:
I am now checking that due date is not None, if it is it is just ignored